tests/test_schema.py

```diff
@@ -290,3 +298,34 @@ def test_schema_validate_json_schema_with_bot_uri_override(tmp_path):
             os.environ["CONDA_SMITHY_BOT_SCHEMA_URI"] = old_val
         else:
             del os.environ["CONDA_SMITHY_BOT_SCHEMA_URI"]
+
+
+@pytest.mark.xfail(
+    reason=(
+        "rattler-build-conda-compat makes global modifications to ruamel.yaml"
+        " - see https://github.com/prefix-dev/rattler-build-conda-compat/issues/88"
+    ),
+)
+def test_schema_with_rattler_build_conda_compat():
```

Comment:
Do I read this test correctly that real-life feedstocks using `random_fraction_to_keep` will remain broken until rbcc is fixed?